src/OidcClient/AuthorizeClient.cs

```diff
@@ -114,8 +114,8 @@ public async Task<AuthorizeState> CreateAuthorizeStateAsync(Parameters frontChan
                 {
                     _logger.LogError("Failed to push authorization parameters");
 
-                    state.Error = "Failed to push authorization parameters";
-                    state.ErrorDescription = parResponse.Error;
+                    state.Error = parResponse.Error;
+                    state.ErrorDescription = "Failed to push authorization parameters";
```

Comment:
Yes, and maybe also:

`state.ErrorDescription = "Failed to push authorization parameters" + parResponse.ErrorDescription;`

of course with some better formatting and null checking, etc

Reply:
The par response only has a single error property, inherited from the base class in IdentityModel